src/main/java/store/reduct/client/ReductClient.java

```diff
@@ -92,14 +98,15 @@ public Bucket createBucket(String name, BucketSettings bucketSettings)
 																									// settings as null
 																									// until invoke
 																									// read.
-		if (httpResponse.statusCode() == 200) {
+		if (httpResponse.statusCode() == 200 || Boolean.TRUE.equals(bucketSettings.getExists())) {
```

Comment:
`Boolean.TRUE.equals(bucketSettings.getExists()))` is this really needed!? 

why not: 
```java
if (httpResponse.statusCode() == 200 || bucketSettings.getExists())
```

Reply:
bucketSettings.getExists() return a boxed boolean - Boolean. All boxed base classes could be as null.
httpResponse.statusCode() == 200 || bucketSettings.getExists() will throw a NullPointerException if httpResponse.statusCode() is not equal to 200